Rumi blog post (sc-2040)

tl6kk opened the pull request.

rebeccabilbro commented on content/en/author/tianshu-li.md:
```diff
@@ -0,0 +1,13 @@
+---
+
+
+Data scientist and NLP specialist with a PhD in Systems Engineering and an enduring curiosity for learning, open source, and new culinary adventures.
```
Feel free to edit 😉  just wanted to make sure we gave a hint into your personality here!

rebeccabilbro reviewed (approved):
This looks great @tl6kk! I've done some light editing and moved the images to the new media folder. I also added in a bio line for you, but you should feel free to modify  it 😁 

I'll plan to publish this sometime next week, sound good?

tl6kk commented:
> This looks great @tl6kk! I've done some light editing and moved the images to the new media folder. I also added in a bio line for you, but you should feel free to modify it 😁
> 
> I'll plan to publish this sometime next week, sound good?

Thank you so much for your edits. They all look great to me. Next week sounds great, so we have weekly blog posts now😉
